src/Handlers/Handler.php

```diff
@@ -24,14 +26,14 @@ public function __construct(
         string $table,
         string $field,
         mixed $input,
-        string $primaryKey = 'id',
+        string $seed = 'scrubber',
+        ?DatabaseKey $primaryKey = null,
         ?string $type = null,
-        string $seed = 'scrubber'
```

Comment:
The order of parameters here didn't seem to match the DataHandler interface, so I've switched them around to match.

Reply:
Good spot, I have moved the `seed` parameter back to the end of the call as I feel it's less related to the database and is just for faker data. Plus I think having the `primaryKey` as the first optional parameter makes more sense.